cumulus/parachains/runtimes/bridge-hubs/bridge-hub-rococo/tests/tests.rs

```diff
@@ -714,4 +716,117 @@ mod bridge_hub_bulletin_tests {
 			false,
 		)
 	}
+
+	#[test]
+	fn location_conversion_works() {
```

Comment:
nit: this test should be in tests root, not under `mod bridge_hub_bulletin_tests`
(move it out of `mod bridge_hub_bulletin_tests`)

Reply:
done